CONTRIBUTING.md

```diff
@@ -58,13 +55,13 @@ Also, remove the "wrench" icon (🔧) from the TOC and the file heading, if appl
 
 If your PR is addressing an existing issue, add the `Fixes #Issue_Number` keyword to the commit message or PR description, so the issue can be automatically closed when the PR is merged. For more information, see [Closing issues via commit messages](https://help.github.com/articles/closing-issues-via-commit-messages/).
 
-The .NET team will review your PR and let you know if the change looks good or if there are any other updates/changes necessary in order to approve it.
+The .NET team will review your PR and let you know if there are any other updates/changes necessary in order to approve it.
 
-**Step 6:** Make any necessary updates to your branch as discussed with the team. 
+**Step 6:** Terspond to the PR feedback.
 
-The maintainers will merge your PR into the master branch once feedback has been applied and your change looks good. 
+The maintainers will merge your PR into the master branch once feedback has been applied and your PR is approved.
 
-On a certain cadence, we push all commits from master branch into the live branch and then you'll be able to see your contribution live at https://docs.microsoft.com/dotnet/. 
+We typically push all commits from master branch into the live branch. You'll be able to see your contribution live at https://docs.microsoft.com/dotnet/ within an hour of the contribution being merged.
```

Comment:
got it. But the original doesn't make much sense and seems to imply quick viewing .